pkg/network/state_sync.go

```diff
@@ -9,12 +10,15 @@ import (
 // StateSync represents state sync module.
 type StateSync interface {
 	AddMPTNodes([][]byte) error
-	bqueue.Blockqueuer
+	bqueue.Blockqueuer[bqueue.Queueable]
```

Comment:
Ditto for statesync, `bqueue.Blockqueuer[bqueue.Queueable]` is now queue-related. Create one more interface with specific methods and reuse it for statesync and for Ledger.